orttraining/orttraining/training_ops/cuda/optimizer/adam.h

```diff
@@ -38,6 +39,7 @@ class AdamOptimizer final : public CudaKernel {
     info.GetAttrOrDefault("beta", &beta_, 0.999f);
     info.GetAttrOrDefault("lambda", &lambda_, 0.0f);
     info.GetAttrOrDefault("epsilon", &epsilon_, 1e-8f);
+    ORT_ENFORCE(info.GetAttr<int64_t>("do_bias_correction", &do_bias_correction_).IsOK(), "Missing/Invalid do_bias_correction");
```

Comment:
>do_bias_correction_ [](start = 61, length = 19)

constructor is a good place to convert to bool

Reply:
Sure.